src/Tarcog/tst/units/GapLayerAtEdgeForcedVentilationWithSolarRadiation.unit.cpp

```diff
@@ -0,0 +1,182 @@
+#include <memory>
+#include <gtest/gtest.h>
+
+#include "WCETarcog.hpp"
+
+class TestGapLayerAtEdgeForcedVentilationWithSolarRadiation : public testing::Test
+{
+private:
+    std::shared_ptr<Tarcog::ISO15099::CSingleSystem> m_TarcogSystem;
+
+protected:
+    void SetUp() override
+    {
+        /////////////////////////////////////////////////////////
+        // Outdoor
+        /////////////////////////////////////////////////////////
+        auto airTemperature = 255.15;   // Kelvins
+        auto airSpeed = 5.5;            // meters per second
+        auto tSky = 255.15;             // Kelvins
+        auto solarRadiation = 700.0;
```

Comment:
@bbuenoun In this test I set the solar radiation to `700`. The computed numbers though do **not** change compared to [the test with solar radiation `0`](https://github.com/ise-bipv/Windows-CalcEngine/pull/2/files#diff-f1510538eafc05022ed4282de85aba1f6ce25356865aa7c328620229546650a7R20).

Reply:
We have to check the (hidden) optical properties of the layers. The absorptivity might be set to zero.

Reply:
The new numbers with about absorptance 10% for the glass layer and absorptance 70% for the shade layer can be seen on the green lines in https://github.com/ise-bipv/Windows-CalcEngine/pull/2/commits/7dc4422ceebcdeb37ac8a5cdca8220a1255693a1#diff-02f7c1e784833ccc1b166b36f337d7a5d836ed05759ab60fa86880260d995ef0

Reply:
Please modify this unit test (let's call it "summer unit test") with the following boundary conditions:
Irradiation: 783 W/m2.
Outdoor temperature: 305.15 K.
Indoor temperature: 297.15 K
Please indicate what are the (hidden) solar reflectance and solar transmittance of the solid and shade layers, respectively.

Reply:
I added the requested test with summer temperatures and irradiation: [GapLayerAtEdgeForcedVentilationWithSolarRadiationInSummer](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/tst/units/GapLayerAtEdgeForcedVentilationWithSolarRadiationInSummer.unit.cpp). I also specified transmittance and emissivity values explicitely in https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/tst/units/GapLayerAtEdgeForcedVentilationWithSolarRadiationInSummer.unit.cpp#L40-L44 and https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/tst/units/GapLayerAtEdgeForcedVentilationWithSolarRadiationInSummer.unit.cpp#L57-L61 and noted how these values are used to calculate the reflectance in [`Surface.cpp#calculateReflectance`](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/Surface.cpp#L115-L126). The implicit defaults used before are defined in [`Layers.hpp#solid`](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/Layers.hpp#L22-L27) and [`Layers.hpp#shading`](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/Layers.hpp#L34-L42).

Reply:
You specified the infrared transmittance (solidLayerFrontIRTransmittance), but we need the SolarTransmittance (the variable might be called "solidLayerFrontSolarTransmittance"), but I'm not sure

Reply:
There are no other arguments to specify the transmittance that I could find. May it be that the argument given to [the helper function `solid`](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/Layers.cpp#L14-L26) as well as to [the constructor of solid layers](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/IGUSolidLayer.cpp#L41-L55) have the wrong name (I used the names of those arguments to name my variables in the tests)? In [the helper function `shading`](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/Layers.cpp#L63-L93) the arguments used in the very same way are named `frontTransmittance` and `backTransmittance` (these arguments are in the case that the shade layer is closed passed to the helper function `solid` as `...IRTransmittance`; so at least here is an inconsistency in the usage). In the case that the shade layer is open, the transmittance arguments are in both usages passed to [the constructor of `CSurface`](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/src/Surface.cpp#L28-L37) in which the argumet is named `t_Transmittance`.

Reply:
Ok. We need to discuss with Simon Vadanovic how the shortwave radiation problem is solved in Wincalc and what is the meaning of "solidLayerSolarAbsorptance", whether it already takes into account what is transmitted from the first layer.

Reply:
The results of [GapLayerAtEdgeForcedVentilationWithSolarRadiationInSummer](https://github.com/ise-bipv/Windows-CalcEngine/blob/add-gaps-with-forced-ventilation/src/Tarcog/tst/units/GapLayerAtEdgeForcedVentilationWithSolarRadiationInSummer.unit.cpp) are plausible